pom.xml

```diff
@@ -193,34 +192,23 @@
 		<dependency>
 			<groupId>org.apache.commons</groupId>
 			<artifactId>commons-lang3</artifactId>
-			<version>3.7</version>
-		</dependency>
-		<dependency>
-			<groupId>org.jvnet.jaxb2_commons</groupId>
-			<artifactId>jaxb2-basics-tools</artifactId>
```

Comment:
completely removed this dependency, so we don't need some obscure third party libraries